src/modules/add.js

```diff
@@ -0,0 +1,77 @@
+import { call, removes, norm } from './edit.js';
+import { status } from './status.js';
+// here is where i created the to do list functions
+export default class ToDoInfo {
+  // created constructor
+  constructor(description, ind) {
+    this.description = description;
+    this.completed = false;
+    this.index = ind;
+  }
+
+  static Add = (description) => {
+    ToDoInfo(description);
+  };
+
+  // created the add function to html document
+  static addToList = (list) => {
+    const mission = document.getElementById('mission');
```

Comment:
It will be better to declare the variable outside the function or at the beginning of the file.